specs/market-api.yaml

```
@@ -641,44 +727,93 @@ paths:
           to return to the Negotiation phase by sending a new Proposal.
 
         * `Cancelled` - Indicates that the Requestor himself has called
-         `cancelAgreement`, which effectively stops the Agreement handshake.
-
+          `cancelAgreement`, which effectively stops the Agreement handshake.
       responses:
-        '200':
+        200:
           description: Agreement approval result.
           content:
             application/json:
               schema:
                 type: string
                 enum: [Approved, Rejected, Cancelled]
-        '401':
-          $ref: '#/components/responses/Unauthorized'
-        '404':
-          $ref: '#/components/responses/NotFound'
-        '408':
-          $ref: '#/components/responses/TimedOut'
-        '409':
+        401:
+          $ref: 'common.yaml#/responses/Unauthorized'
+        404:
+          $ref: 'common.yaml#/responses/NotFound'
+        408:
+          $ref: 'common.yaml#/responses/Timeout'
+        409:
           description: Agreement not in `Pending` state.
-        '410':
-          $ref: '#/components/responses/Expired'
+        410:
+          $ref: 'common.yaml#/responses/Gone' # Expired
         default:
-          $ref: '#/components/responses/UnexpectedError'
+          $ref: 'common.yaml#/responses/UnexpectedError'
+
+  /agreements/events:
+    parameters:
+      - $ref: 'common.yaml#/parameters/pollTimeout'
+      - $ref: 'common.yaml#/parameters/afterTimestamp'
+      - $ref: 'common.yaml#/parameters/maxEvents'
+      - $ref: 'common.yaml#/parameters/appSessionId'
```

To enable querying only for events regarding single agreement
```suggestion
      - $ref: 'common.yaml#/parameters/appSessionId'
      - $ref: 'common.yaml#/parameters/queryAgreementId'
```